static/scss/_variables.scss

```diff
@@ -1,14 +1,14 @@
 $mit-red: #a31e31;
-$body-bg-color: #f5f5f5;
+$body-bg-color: #fff;
 $linkcolor: #2568be;
 $icon-color: #007de1;
 $icon-hover: #30a2fd;
 $fontgray: #666;
 $card-bg-color: #fff;
 $card-shadow: #c7c7c7;
-$tool-icon-color: #007de1;
-$tool-icon-color-hover: #30a2fd;
-$dropbox-blue: #007ee5;
+$tool-icon-color: #ababab;
+$tool-icon-color-hover: #007de1;
+$blue-button: #007ee5;
```

Comment:
We can leave this, but just FYI this is the hex value that the dropbox branding guide recommended, hence the "dropbox-blue" class name